homeassistant/components/rfxtrx/__init__.py

```diff
@@ -520,6 +521,18 @@ def device_info(self):
             "model": self._device.type_string,
         }
 
+    def _event_applies(self, event, device_id):
+        """Check if event applies to me."""
+        if "Command" in event.values and event.values["Command"] in COMMAND_GROUP_LIST:
+            # A group event with device id 213c7f2:1 applies to all devices with ids 213c7f2:n.
+            if ":" not in event.device.id_string:
+                return False
```

Comment:
You should be able to use python partition function here instead.
http://python-reference.readthedocs.io/en/latest/docs/str/partition.html

group_id, _, device_id = partition(event.device.id_string, ":")
if self.group_id == group_id...

Then have the group_id of the entity a precalculated property.

Reply:
Python is not my native language, so thanks for the tip!